scripts/benchmark_all.sh

```diff
@@ -111,8 +122,16 @@ do
       ( "${build_folder}"/"$benchmark" --scheduler --clients 1 --cores ${num_phy_cores} -m Ordered -r ${runs} -w ${warmup_seconds} -o "${build_folder}/benchmark_all_results/${benchmark}_${commit}_mt_ordered.json" 2>&1 ) | tee "${build_folder}/benchmark_all_results/${benchmark}_${commit}_mt_ordered.log"
     fi
 
-    echo "Running $benchmark for $commit... (multi-threaded, shuffled, $num_phy_cores clients)"
-    ( "${build_folder}"/"$benchmark" --scheduler --clients ${num_phy_cores} --cores ${num_phy_cores} -m Shuffled -t ${mt_shuffled_runtime} -o "${build_folder}/benchmark_all_results/${benchmark}_${commit}_mt.json" 2>&1 ) | tee "${build_folder}/benchmark_all_results/${benchmark}_${commit}_mt.log"
+    if [ "$benchmark" = "hyriseBenchmarkTPCC" ]; then
+      # We test two scenarios: high and normal contention (see above, each for 10 minutes).
+      echo "Running $benchmark for $commit... (multi-threaded, shuffled, high contention, $clients_tpcc_high_contention clients, $tpcc_warehouse_count_high_contention warehouses)"
+      ( "${build_folder}"/"$benchmark" --scheduler --clients ${num_phy_cores} --cores ${num_phy_cores} -s $tpcc_warehouse_count_high_contention -m Shuffled -t ${tpcc_shuffled_runtime} -o "${build_folder}/benchmark_all_results/${benchmark}_${commit}_mt.json" 2>&1 ) | tee "${build_folder}/benchmark_all_results/${benchmark}_${commit}_mt.log"
+      echo "Running $benchmark for $commit... (multi-threaded, shuffled, low contention, $clients_tpcc_low_contention clients, $tpcc_warehouse_count warehouses)"
+      ( "${build_folder}"/"$benchmark" --scheduler --clients ${num_phy_cores} --cores ${num_phy_cores} -s ${tpcc_warehouse_count} -m Shuffled -t ${tpcc_shuffled_runtime} -o "${build_folder}/benchmark_all_results/${benchmark}_${commit}_mt.json" 2>&1 ) | tee "${build_folder}/benchmark_all_results/${benchmark}_${commit}_mt.log"
```

Comment:
Use different names for the output files and add them to the configs below so they are actually printed?